articles/iot-edge/how-to-visual-studio-develop-module.md

```diff
@@ -158,6 +158,8 @@ To initialize the tool, provide an IoT Edge device connection string from IoT Hu
 
 Typically, you'll want to test and debug each module before running it within an entire solution with multiple modules.
 
+[!TIP] Make sure you have switched over to the correct Docker container mode (i.e. Linux Container Mode or Windows Container Mode) depending on the type of IoT Edge Module you are developing. From the Docker Desktop menu, you can toggle between the two types of modes. Select Switch to Windows containers to use Windows containers, or select Switch to Linux containers to use Linux containers. 
```

Comment:
Thanks @AndresPad. I suggested some formatting edits to make and then I can sign off on these changes:

```suggestion
>[!TIP]
>Make sure you have switched over to the correct Docker container mode, either Linux container mode or Windows container mode, depending on the type of IoT Edge module you are developing. From the Docker Desktop menu, you can toggle between the two types of modes. Select **Switch to Windows containers** to use Windows containers, or select **Switch to Linux containers** to use Linux containers. 
```